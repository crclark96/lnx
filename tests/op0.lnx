rax + rcx;
bl - al;
edx = 102;
dx = cx;
ecx ^ edx;
ebx ^ 0xde123;
