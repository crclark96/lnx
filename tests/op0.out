add rax, rcx
sub bl, al
mov edx, 102
mov dx, cx
xor ecx, edx
xor ebx, 0xde123
